x-pack/filebeat/input/cel/input.go

```diff
@@ -896,8 +906,20 @@ func newProgram(ctx context.Context, src, root string, client *http.Client, limi
 	return prg, nil
 }
 
-func evalWith(ctx context.Context, prg cel.Program, state map[string]interface{}) (map[string]interface{}, error) {
-	out, _, err := prg.ContextEval(ctx, map[string]interface{}{root: state})
+func evalWith(ctx context.Context, prg cel.Program, state map[string]interface{}, now time.Time) (map[string]interface{}, error) {
```

Comment:
Optional idea: this `now` parameter could be called `evalStart` to help emphasize and explain the change of value done in the shadowing.

Reply:
I think `now` is more evocative of what I mean here.